examples/java/src/main/java/org/apache/beam/examples/complete/kafkatopubsub/kafka/consumer/Utils.java

```diff
@@ -162,4 +165,11 @@ public static boolean isSslSpecified(KafkaToPubsubOptions options) {
         || options.getKeystorePath() != null
         || options.getKeyPassword() != null;
   }
+
+  public static Map<String, Object> parseKafkaConsumerConfig(String kafkaConsumerConfig) {
+    return Arrays.stream(kafkaConsumerConfig.split(";"))
+        .map(s -> s.split("="))
+        .map(kv -> Pair.of(kv[0], kv[1]))
+        .collect(Collectors.toMap(Pair::getKey, Pair::getValue));
```

Comment:
nit: If I were writing this I would probably combine these two lines and avoid creating the `Pair`:
```suggestion
        .collect(Collectors.toMap(kv -> kv[0], kv -> kv[1]));
```
If you prefer it with the Pair that's fine too.

One thing I think we _should_ change is make this a private static method in `KafkaToPubsub`, since it's only user there.

Reply:
Removed `Pair` related code, but left method in place.
This class is created for such methods, in my opinion it is better to declare them in `Utils` in order to leave main class as clean as possible